schemas/test/en/test_first_non_empty_item_cross_section_dependencies.json

```diff
@@ -0,0 +1,376 @@
+{
```

Comment:
This would be better if it's hub type schema with first section required to see the hub, atm it is not clear when you enter second section until you are on the summary.

Reply:
Changed to a Hub 👍

Reply:
I have also changed the schema a bit so there shouldnt be a total turnover question in the first section. Just asks for the date